### PR TITLE
Walking skeleton

### PR DESCRIPTION
**Pull request by whaley-ethan**
integrated with travis CI, built walking skeleton

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ## Tic Tac Toe
 This is an implementation on the command line of the classic game Tic Tac Toe written in Java
 ## Build Status
-todo: implement CI, put status here
+[![Build Status](https://travis-ci.com/solidlikeplato/tic-tac-toe-java.svg?branch=master)](https://travis-ci.com/solidlikeplato/tic-tac-toe-java)
```
Comment (jchung722) [inline readme.md:4]: nice 🔥

File: .settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1
```
Comment (jchung722) [inline .settings/org.eclipse.buildship.core.prefs]: I think some of these folders/files can go into .gitignore; if you're using the eclipse IDE, you can find a specific gitignore for it online

File: readme.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ## Tic Tac Toe
```
Comment (jchung722) [inline readme.md:1]: Looks good! I think the only thing missing is a list of dependencies & setup/installation instructions (if there are any yet)

**Review by jchung722 — CHANGES_REQUESTED** [review]
Looking good! Just a couple comments

File: readme.md (hunk 3 above)
Comment (whaley-ethan) [reply readme.md:1]: Changes made. Should I put in an empty Setup/Installation section as a stub or just add it once it becomes relevant?

Comment (jchung722) [reply readme.md:1]: I'm okay with adding once it becomes relevant; not sure what everyone else thinks

**Review by jchung722 — APPROVED** [review]

**Pull request merged**